requirements/edx/base.in

```diff
@@ -47,7 +47,7 @@ django-filter==1.0.4                # Allows users to filter Django querysets dy
 django-ipware                       # Get the client's real IP address
 django-method-override
 django-model-utils
-django-mptt>=0.8.6,<0.9
+django-mptt>=0.10.0
```

Comment:
why are you adding > condition ?

Reply:
I added the condition so it can automatically get the updated version whenever a new version is available.